rpcs3/Emu/RSX/D3D12/D3D12GSRender.cpp

```diff
@@ -278,6 +278,7 @@ D3D12GSRender::~D3D12GSRender()
 
 void D3D12GSRender::on_exit()
 {
+	return GSRender::on_exit();
```

Comment:
It must clarify this is the last (tail) call and other actions must be placed before.